lib/internal/streams/pipeline.js

```diff
@@ -353,7 +353,7 @@ function pipe(src, dst, finish, { end }) {
     }
   });
 
-  src.pipe(dst, { end });
+  src.pipe(dst, { end: false }); // If end is true we already will have a listener to end 'dst'.
```

Comment:
```suggestion
  src.pipe(dst, { end: false }); // If end is true we already will have a listener to end dst.
```

Nit to not confuse it with an event.

Reply:
done!